### DISCUSSION TITLE
How to use pip inside venv?

### DISCUSSION THREAD

**Discussion by itsankitkp** [RustPython/RustPython#4904] [Q&A]
Steps tried:
1. cargo build --release --features ssl,jit
2. target/release/rustpython -m venv venv
3. venv/bin/activate
4. rustpython --install-pip get-pip
5. pip install guincorn

```
 pip3 install gunicorn
Traceback (most recent call last):
  File "/home/ankit/RustPython/venv/bin/pip3", line 5, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip'
```

Different variant of #4 tried is `target/release/rustpython -u --install-pip ensurepip --user`, `venv/bin/rustpython   --install-pip` (as per as docs)
for #5 `pip3 install gunicorn`. Also tried it directly
```
 venv/bin/rustpython -m pip install gunicorn
/home/ankit/RustPython/venv/bin/rustpython: No module named pip
```

Not sure what I am missing.

**Comment by youknowone** [comment]
I found it doesn't work. This is a bug.
